io-engine/src/bin/io-engine-client/v1/snapshot_cli.rs

```diff
@@ -202,24 +202,32 @@ async fn create_for_nexus(
         })?
         .to_owned();
     let replica_uuid = matches
-        .get_many::<String>("replica_uuid")
+        .get_one::<String>("replica_uuid")
```

Comment:
Any reason to move from many to one?
I guess this means all uuids must be provided with 1 string?

Reply:
It is changed to map each replica_uuid to snapshot_uuid. Both will be given as a single string as shown below.

run-io-engine snapshot create_for_nexus a4d9bbd8-ed98-4905-81cd-f2ff004eff06 e1 t1 snap1 "a0e7832c-6f46-461d-a8c5-74cdf4ddc53f,c2b46c53-2bbe-4286-949d-47f1859c9eac,6c9cdf49-7261-4fdf-81af-1e82949fc786" "15b41216-2fc6-4b4b-90e4-6d772d49bdb1,15b41216-2fc6-4b4b-90e4-6d772d49bdb2,15b41216-2fc6-4b4b-90e4-6d772d49bdb3"

Reply:
Oh, I see now, because the arguments are positional